### DISCUSSION TITLE
Assumptions on where to find input .mps files

### DISCUSSION THREAD

**Discussion by spkelle2** [ambros-gleixner/MIPcc23#19] [Q&A] [answered]
Since our competition files will be executed as `sh mipcomp.sh dataset.test`, could someone point me to where I could find more information on the following:

1. Will the file paths in `dataset.test` be fully qualified? If not, what directory should we assume each path to exist in?
2. Will the file paths in `dataset.test` have `.gz` extensions or can we assume them to be unzipped at run time?

**Comment by ambros-gleixner** [comment, marked as answer]
Thanks for the question!
1. Yes, you can assume paths will start from `/`.
2. On the evaluations server, we will provide all instances as unzipped `mps` files.

**Reply by spkelle2** [reply]
Perfect! That was the answer I was hoping for. Thank you for getting back to me to let me know.

**Reply by spkelle2** [reply]
@ambros-gleixner I have a related question on this topic. When you call `sh mipcomp.sh dataset.test`, are you going to be moving the `.sh` file to the folder where the `.test` file is, moving the `.test` file to the folder where the `.sh` file is, or will one of them be called from an absolute path?

**Reply by ambros-gleixner** [reply]
We will call your `mipcomp.sh` from your directory, and `dataset.test` will be an absolute path.

Feel free to ping us when you have a running pre-version and we can check whether everything works for us.

**Reply by spkelle2** [reply]
you guys make this easy. Thank you!
